draft-ietf-quic-transport.md

```diff
@@ -205,28 +205,28 @@ Out-of-order packet:
 
 Endpoint:
 
-: An entity that can participate in a QUIC connection by generating,
-  receiving, and processing QUIC packets. There are only two types of endpoint
-  in QUIC: client and server.
+: An entity that can participate in a QUIC connection by generating, receiving,
+  and processing QUIC packets. There are only two types of endpoint in QUIC:
+  client and server.
 
 Client:
 
 : The endpoint initiating a QUIC connection.
 
 Server:
 
-: The endpoint accepting incoming QUIC connections.
+: The endpoint accepting an incoming QUIC connection.
 
 Address:
 
-: When used without qualification, the tuple of IP version, IP address, UDP
-  protocol, and UDP port number that represents one end of a network path.
+: When used without qualification, the tuple of IP version, IP address, and UDP
+  port number that represents one end of a network path.
```

Comment:
This is supposed to say that the IP protocol number indicates UDP. We discussed this when adding the text, and couldn't come up with a better way to say it.

Reply:
QUIC is defined (per invariants) to run over UDP.  I don't think including a constant in the definition of the tuple gains you any uniqueness, and stating that it's a UDP port already emphasizes what the protocol has to be.

Reply:
I think that Mike's edit is fine.  We can ignore the fixed parts of this, which includes that it is QUIC.